### PR TITLE
Source install

### PR DESCRIPTION
**Pull request by ualaska-se**

### CODE REVIEW

File: libraries/python_install_helper.rb [hunk 1]
```diff
@@ -31,77 +45,77 @@ def rel_path_to_python_binary(new_resource)
       return "bin/python#{revision}"
```
Comment (Arsh25) [inline libraries/python_install_helper.rb:45]: Needs to be an absolute directory. Missing leading /

**Review by Arsh25 — CHANGES_REQUESTED** [review]

File: libraries/python_install_helper.rb (hunk 1 above)
Comment (ualaska-se) [reply libraries/python_install_helper.rb:45]: For a function named rel_path...? :)

File: libraries/python_install_helper.rb [hunk 2]
```diff
@@ -1,9 +1,23 @@
...
-  module Helper
+  # This module exposes helpers to the client
+  module Public
+    def default_python_version
+      return '3.7.4'
+    end
+
+    def default_python_directory
+      # Must match source_install
+      return "opt/python/#{default_python_version}"
```
Comment (Arsh25) [inline libraries/python_install_helper.rb]: Should this be absolute?

Comment (ualaska-se) [reply libraries/python_install_helper.rb]: Yeah, I knew what you meant:)

**Review by Arsh25 — APPROVED** [review]

**Pull request merged**